.env.development

```
@@ -9,13 +9,14 @@ CHECKOUT_POLLING_ACTIVATION_ATTEMPTS=3
 CHECKOUT_ENV=develop
 CHECKOUT_RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
 CHECKOUT_DONATIONS_URL=https://api.uat.platform.pagopa.it/donations/api/v1/availabledonations
-CHECKOUT_ECOMMERCE_HOST=http://localhost:1234
+CHECKOUT_ECOMMERCE_HOST=https://api.dev.platform.pagopa.it
```

```suggestion
CHECKOUT_ECOMMERCE_HOST=http://localhost:1234
```
For env development leaves pointing to localhost

---

it is, that was a miscommit